scripts/postgres/create_database_compstore-postgres.sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE "compstore-postgres";
+\c "compstore-postgres";
```

Comment:
Oprócz stworzenia naszej bazy usunąłem domyślną "postgres"

Reply:
Zrozumiałem to tak jakbyś usunął bazę `postgres`, a Ty tylko zmieniłeś domyślną na `compstore-postgres`.